app/donglet/app-g030.toml

```diff
@@ -6,8 +6,8 @@ board = "donglet-g030"
 
 [kernel]
 name = "app-donglet"
-requires = {flash = 11702, ram = 1536}
-features = ["g030", "panic-halt"]
+requires = {flash = 18000, ram = 1328}
```

Comment:
Intentional change?

Reply:
(Answered this in chat, but, yes, the g0 was using `panic-halt` before, so it takes a flash hit to bring in formatting code.)